src/CalcManager/Ratpack/logic.cpp

```diff
@@ -16,7 +16,7 @@
 #include "pch.h"
 #include "ratpak.h"
 
-
+using namespace std;
```

Comment:
Honestly, this is awful IMO.

Reply:
Some people like it and some don't.  The simple answer here is that this is consistent with the rest of the codebase.  If the community wants to have a discussion about changing this precedent, someone can make an issue and we can have an open conversation.